docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -346,6 +439,76 @@ Before executing an action, TestCafe waits for the target element to appear
 in the DOM and become visible. If this does not happen
 within the [selector timeout](#selector-timeout), the test fails.
 
+Note that if a selector initializer has multiple matching DOM nodes on the page, the action will be executed only for the first node from the matching set.
+
+### Define Assertion Actual Value
+
+You can check whether a particular DOM node has an expected state by passing a selector property directly to [test actions](../assertions/index.md).
+
+```js
+
+import { Selector } from 'testcafe';
+
+fixture `My fixture`
+    .page `http://devexpress.github.io/testcafe/example/`;
+
+test('Assertion with Selector', async t => {
+    const developerNameInput = Selector('#developer-name');
+
+    await t
+        .expect(developerNameInput.innerText).eql('')
+        .typeText(developerNameInput, 'Peter')
+        .expect(developerNameInput.innerText).eql('Peter');
+});
```

Comment:
We need a note that in that case property will not be calculated immediately, but only then assertion will be executed.